lib/regex.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
hm.. actually, why move it to a separate entire module?

and also needs a 'use strict'

Reply:
Oh, I see why module. But still needs 'use strict'

Reply:
I extracted it into a new module because I wanted to add a unit test for it.

Reply:
I didn't want to mess with the exports of the hyperswitch module.

Reply:
I tried to find the trick you had to expose a function or variable for unit testing only but wasn't lucky enough.